<internal_error>
<sub_system>ERR</sub_system>
<callstack>
	0x77e8268081e1: ccl_err + 0x81e1 (_ZN15ERR_STACKWALKER15get_stack_traceEPPKviiPv + 0xb1)
	0x77e82680b22a: ccl_err + 0xb22a (_Z14err_terminatorv + 0x4b)
	0x77e882e33506: jtag_client + 0x33506 (_ZN10__cxxabiv111__terminateEPFvvE + 0x6)
	0x77e882e33533: jtag_client + 0x33533
	0x77e882e33686: jtag_client + 0x33686 (__cxa_rethrow + 0x46)
	0x77e866fb5034: QtCore.so.4 + 0x1b5034 (_ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 0x2f4)
	0x77e866fb9c84: QtCore.so.4 + 0x1b9c84 (_ZN16QCoreApplication4execEv + 0xb4)
	   0x401efb: quartus + 0x1efb (_Z8qgq_mainiPPKc + 0x7b)
	0x77e82d040990: ccl_msg + 0x40990 (_Z15msg_main_threadPv + 0x10)
	0x77e825c060cc: ccl_thr + 0x60cc (thr_final_wrapper + 0xc)
	0x77e82d040a4f: ccl_msg + 0x40a4f (_Z18msg_thread_wrapperPFPvS_ES_ + 0x62)
	0x77e826e0a679: ccl_mem + 0xa679 (_Z18mem_thread_wrapperPFPvS_ES_ + 0x99)
	0x77e826809034: ccl_err + 0x9034 (_Z18err_thread_wrapperPFPvS_ES_ + 0x27)
	0x77e825c06492: ccl_thr + 0x6492 (thr_thread_wrapper + 0x15)
	0x77e82d04294c: ccl_msg + 0x4294c (_Z12msg_exe_mainiPPKcPFiiS1_E + 0xa3)
	   0x401fe6: quartus + 0x1fe6 (main + 0x26)
	0x77e81942a1ca: c.so.6 + 0x2a1ca
	0x77e81942a28b: c.so.6 + 0x2a28b (__libc_start_main + 0x8b)
	   0x401d99: quartus + 0x1d99
</callstack>
<error>*** Fatal Error: Unhandled Exception</error>
<date>Thu Nov 21 21:49:23 2024
</date>
<version>Version 19.1.0 Build 670 09/22/2019 SJ Lite Edition</version>
</internal_error>